src/Compilers/CSharp/Test/Emit/CodeGen/IndexAndRangeTests.cs

```diff
@@ -4236,5 +4214,170 @@ static void M(Span<int> s)
             var comp = CreateCompilation(source, targetFramework: TargetFramework.Net70);
             comp.VerifyDiagnostics();
         }
+
+        [Theory]
+        [InlineData(false)]
+        [InlineData(true)]
```

Comment:
The combinatorial data attribute works great for this.
```suggestion
        [CombinatorialData]
```

Reply:
What a great package. I'll use it.